src/components/Work/Tabs/Download.js

```diff
@@ -32,9 +32,9 @@ const WorkTabsDownload = React.memo(function ({ item }) {
   const [currentId, setCurrentId] = useState();
   const [currentLabel, setCurrentLabel] = useState();
 
-  const iiifServerUrl = item.representative_file_url.slice(
+  const iiifServerUrl = item.representativeFileSet.url.slice(
     0,
-    item.representative_file_url.lastIndexOf("/")
+    item.representativeFileSet.url.lastIndexOf("/")
```

Comment:
I keep getting an error trying to test this. 
<img width="1385" alt="Screen Shot 2021-01-11 at 10 15 18 AM" src="https://user-images.githubusercontent.com/72217877/104207710-040db700-53f6-11eb-8ccc-a47c00d6cfad.png">